src/server/managers/AppExternalComponentManager.ts

```diff
@@ -0,0 +1,74 @@
+import { IExternalComponent } from '../../definition/externalComponent';
+
+/**
+ * The external component manager for the Apps.
+ *
+ * An App will register external component during their `initialize` method.
```

Comment:
```suggestion
 * An App will register an external component during their `initialize` method.
```

Reply:
resolved in https://github.com/RocketChat/Rocket.Chat.Apps-engine/pull/149/commits/2a4e5ca641b067de0dd4d609d7f1877478e863d3

Reply:
resolved in https://github.com/RocketChat/Rocket.Chat.Apps-engine/pull/149/commits/b49dcf1ccc902d4c766e671a609cc2b4f4c31678